sqlite.go

```diff
@@ -94,6 +94,13 @@ func (dialector Dialector) ClauseBuilders() map[string]clause.ClauseBuilder {
 				}
 			}
 		},
+		"FOR": func(c clause.Clause, builder clause.Builder) {
+			if _, ok := c.Expression.(clause.Locking); ok {
```

Comment:
I'm not sure if I should check the values is "UPDATE" or "SHARE".
What do you think?